pkg/build/node/cni.go

```diff
@@ -141,8 +141,7 @@ data:
             "type": "host-local",
             "dataDir": "/run/cni-ipam-state",
             "routes": [
-              {"dst": "0.0.0.0/0"},
-              {"dst": "::/0"}
+              {"dst": "0.0.0.0/0"},` + "{{`{{ if .IPv6 }}\n              {\"dst\": \"::/0\"}{{end}}`}}" + `
```

Comment:
I didn´t dig into this but it may possible that we don´t need this field at all since the CNI plugin set the default gw. 
Will investigate and report back

Reply:
it turns out we do need it

Reply:
We should remove this configmap from the installation manifest and embed it into kindnet, this will solve the nesting templating

Reply:
yeah, might do that in a different PR

Reply:
done